environment_maple.yml

```diff
@@ -1,164 +1,13 @@
-  - jpeg=9e=h7f8727e_0
-  - json-c=0.13.1=h1bed415_0
-  - kealib=1.4.7=h77bc034_6
-  - keras=2.3.1=0
-  - keras-applications=1.0.8=py_1
-  - keras-base=2.3.1=py36_0
-  - keras-preprocessing=1.1.2=pyhd3eb1b0_0
-  - kiwisolver=1.3.1=py36h2531618_0
-  - krb5=1.16.1=hc83ff2d_6
-  - lcms2=2.12=h3be6417_0
-  - lerc=3.0=h295c915_0
-  - libboost=1.73.0=h28710b8_12
-  - libcurl=7.61.1=heec0ca6_0
-  - libdap4=3.19.1=h6ec2957_0
-  - libdeflate=1.8=h7f8727e_5
-  - libedit=3.1.20221030=h5eee18b_0
-  - libffi=3.2.1=hf484d3e_1007
-  - libgcc-ng=11.2.0=h1234567_1
-  - libgdal=2.3.2=h9d4a965_0
-  - libgfortran-ng=7.5.0=ha8ba4b0_17
-  - libgfortran4=7.5.0=ha8ba4b0_17
-  - libgomp=11.2.0=h1234567_1
-  - libkml=1.3.0=h096b73e_6
-  - libnetcdf=4.6.1=h10edf3e_2
-  - libpng=1.6.37=hbc83047_0
-  - libpq=10.5=h1ad7b7a_0
-  - libprotobuf=3.17.2=h4ff587b_1
-  - libspatialite=4.3.0a=he475c7f_19
-  - libssh2=1.8.0=h9cfc8f7_4
-  - libstdcxx-ng=11.2.0=h1234567_1
-  - libtiff=4.5.0=h6a678d5_1
-  - libuuid=1.41.5=h5eee18b_0
-  - libwebp-base=1.2.4=h5eee18b_0
-  - libxcb=1.15=h7f8727e_0
-  - libxml2=2.9.14=h74e7548_0
-  - lz4-c=1.9.4=h6a678d5_0
-  - markdown=3.3.4=py36h06a4308_0
-  - matplotlib=3.3.4=py36h06a4308_0
-  - matplotlib-base=3.3.4=py36h62a2d02_0
-  - mkl=2020.2=256
-  - mkl-service=2.3.0=py36he8ac12f_0
-  - mkl_fft=1.3.0=py36h54f3939_0
-  - mkl_random=1.1.1=py36h0573a6f_0
-  - ncurses=6.4=h6a678d5_0
-  - networkx=2.5=py_0
-  - numpy=1.17.4=py36hc1035e2_0
-  - numpy-base=1.17.4=py36hde5b4d6_0
-  - olefile=0.46=py36_0
-  - openjpeg=2.4.0=h3ad879b_0
-  - openssl=1.0.2u=h7b6447c_0
-  - pcre=8.45=h295c915_0
-  - pillow=8.3.1=py36h2c7a002_0
-  - pip=21.2.2=py36h06a4308_0
-  - pixman=0.40.0=h7f8727e_1
-  - poppler=0.65.0=h581218d_1
-  - poppler-data=0.4.11=h06a4308_1
-  - proj4=5.0.1=h14c3975_0
-  - pyparsing=3.0.4=pyhd3eb1b0_0
-  - pyqt=5.9.2=py36h05f1152_2
-  - pyshp=2.1.0=py_0
-  - python=3.6.6=h6e4f718_2
-  - python-dateutil=2.8.2=pyhd3eb1b0_0
-  - pywavelets=1.1.1=py36h7b6447c_2
-  - pyyaml=5.4.1=py36h27cfd23_1
-  - qt=5.9.6=h8703b6f_2
-  - readline=7.0=h7b6447c_5
-  - scikit-image=0.16.2=py36h0573a6f_0
-  - scipy=1.5.2=py36h0b6359f_0
-  - setuptools=58.0.4=py36h06a4308_0
-  - shapely=1.6.4=py36h7ef4460_0
-  - sip=4.19.8=py36hf484d3e_0
-  - six=1.16.0=pyhd3eb1b0_1
-  - sqlite=3.33.0=h62c20be_0
-  - tensorboard=1.14.0=py36hf484d3e_0
-  - tensorflow=1.14.0=gpu_py36h3fb9ad6_0
-  - tensorflow-base=1.14.0=gpu_py36he45bfe2_0
-  - tensorflow-estimator=1.14.0=py_0
-  - tensorflow-gpu=1.14.0=h0d30ee6_0
-  - termcolor=1.1.0=py36h06a4308_1
-  - tk=8.6.12=h1ccaba5_0
-  - toolz=0.11.2=pyhd3eb1b0_0
-  - tornado=6.1=py36h27cfd23_0
-  - typing_extensions=4.1.1=pyh06a4308_0
-  - werkzeug=2.0.3=pyhd3eb1b0_0
-  - wheel=0.37.1=pyhd3eb1b0_0
-  - wrapt=1.12.1=py36h7b6447c_1
-  - xerces-c=3.2.4=h94c2ce2_0
-  - xz=5.2.10=h5eee18b_1
-  - yaml=0.2.5=h7b6447c_0
-  - zipp=3.6.0=pyhd3eb1b0_0
-  - zlib=1.2.13=h5eee18b_0
-  - zstd=1.5.2=ha4553b6_0
+  - python=3.9
```

Comment:
@kaylahardie and I were just talking and we were actually thinking to bump this up to python 3.10 to buy us more time in terms of LTS. I don't expect the transition from 3.9 to 3.10 to be too bad but lmk if you would like me to take care of that!

Reply:
Please also update this one Todd.

Reply:
updated to 3.10 as well. 

I checked that this installed right and ran within a Docker container.